CHANGES.txt

```diff
@@ -1,6 +1,9 @@
 Changelog
 ---------
 
+1.2
+ Include softPurgeKeys to allow batch call to purge by multiple surrogate keys.
```

Comment:
What if we include the limit? 

From the docs: Up to 256 surrogate keys can be purged in one batch request.

Reply:
done